clang/test/CXX/drs/cwg18xx.cpp

```diff
@@ -206,19 +206,28 @@ namespace cwg1814 { // cwg1814: yes
 #endif
 }
 
-namespace cwg1815 { // cwg1815: no
+namespace cwg1815 { // cwg1815: 19
```

Comment:
This should be `// cwg1815: 20` I think?

Reply:
Seems to be the case, yeah.
From what I see, this effort was started back before 19 has branched, and was never backported to 19 branch.

Reply:
Oh, I missed this! I'll fix it. Thank you for point out this.